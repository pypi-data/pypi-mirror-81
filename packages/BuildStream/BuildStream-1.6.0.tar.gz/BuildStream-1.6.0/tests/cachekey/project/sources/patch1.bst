kind: import
sources:
- kind: patch
  path: files/patches/patch.diff
