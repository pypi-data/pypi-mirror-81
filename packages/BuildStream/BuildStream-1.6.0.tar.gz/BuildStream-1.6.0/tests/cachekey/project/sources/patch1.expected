4408a9652884249b5b709d8defd726d3f5aa3279418fd303180bc12d084172ef